backend/apollo/elections/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
Trying to write this comment for a third time (thx GitHub)

So about testing - we'll go with `pytest`

An example test:
```python
from pytest import mark
from rest_framework.test import APIClient
from rest_framework.reverse import reverse
from rest_framework import status
from apollo.elections.models import Election

def _create_election(election_data, user):
    api_client = APIClient()  # may be moved to a fixture in the future
    api_client.force_authenticate(user=user)
    return api_client.post(reverse("elections:election-list"), data=election_data, format="json")

@mark.django_db
def test_create_election():
    election_data = { ... }  # if used more than once then also can be moved to a fixture
    user = UserFactory()  # not sure if we have factories ATM. If not then just use `User.objects.create`. Also can be moved to a fixture
    response = _create_election(election_data, user)
    assert response.status_code == status.HTTP_201_CREATED
    assert Election.objects.last() == ...  # verify that the election was properly instantiated
```

The things that we want to test include all the operations exposed by the API, e.g. instantiating/updating/deleting/getting various things (both successful and unsuccessful due to malformed data (400), lack of permissions (403) or lack of authorization (401)).

Reply:
Btw, we don't need to test _all of that_ right now, especially given the fact that we're not yet sure about the acceptance criteria we're interested in. IMO, for now it's sufficient to just test that we can create and retrieve the elections.